Packs/DevSecOps/Integrations/MinIO/MinIO.yml

```diff
@@ -79,87 +79,122 @@ script:
-    - description: Object Name
+    - description: Object Name.
       name: name
       required: true
     description: Remove an object.
     name: minio-remove-object
     outputs:
     - contextPath: MinIO.Objects.name
-      description: Object Name
+      description: Object Name.
       type: Unknown
     - contextPath: MinIO.Objects.status
-      description: Object Status
+      description: Object Status.
       type: Unknown
   - arguments:
-    - description: Bucket Name
+    - description: Bucket Name.
       name: bucket_name
-      predefined:
-      - ''
       required: true
-    - description: File Entry ID
+    - description: File Entry ID.
       name: entry_id
       required: true
-    - description: File Type
+    - description: File Type.
       name: content_type
     - description: Any additional metadata to be uploaded along with your PUT request.
       name: metadata
     description: Uploads data from a file to an object in a bucket.
     name: minio-fput-object
   - arguments:
-    - description: Bucket Name
+    - description: Bucket Name.
       name: bucket_name
-      predefined:
-      - ''
       required: true
-    - description: Contains object data
+    - description: Contains object data.
       name: data
       required: true
-    - description: Object name in the bucket
+    - description: Object name in the bucket.
       name: name
       required: true
-    - description: File Type
+    - description: File Type.
       name: content_type
     - description: Any additional metadata to be uploaded along with your PUT request.
       name: metadata
     description: Uploads data from a stream to an object in a bucket.
     name: minio-put-object
+  - name: minio-copy-object
+    arguments:
+    - name: bucket_name_src
+      description: Bucket Name Source.
+      required: true
+    - name: name_src
+      description: Object Name Source.
+      required: true
+    - name: bucket_name_dst
+      description: Bucket Name Destination.
+      required: true
+    - name: name_dst
+      description: Object Name Destination.
+      required: true
+    - name: metadata
+      description: Any additional metadata to be copied
+    description: Create an object by server-side copying data from another object.
+  - name: minio-get-tags
+    arguments:
+    - name: bucket_name
+      description: Bucket Name.
+      required: true
+    - name: name
+      description: Object Name.
+      required: true
+    description: Get tags configuration of an object.
+  - name: minio-set-tag
+    arguments:
+    - name: bucket_name
+      description: Bucket Name.
+      required: true
+    - name: name
+      description: Object Name.
+      required: true
+    - name: tag_key
+      description: Key for the tag.
+      required: true
+    - name: tag_value
+      description: Value for the tag.
+      required: true
+    description: Set tags configuration to an object.
   dockerimage: demisto/minio:1.0.0.19143
```

Comment:
see the [validation error ](https://app.circleci.com/pipelines/github/demisto/content/309045/workflows/d1558365-d3a2-47eb-9925-cf5cd8c4462d/jobs/723810?invite=true#step-119-4918_163) about this docker image